src/common/wrapsizer.cpp

```diff
@@ -469,7 +469,7 @@ void wxWrapSizer::CalcMinFromMinor(int totMinor)
     }
 
     // Now have min size in the opposite direction
-    m_calculatedMinSize = SizeFromMajorMinor(lineSize,sumMinor);
+    m_calculatedMinSize = SizeFromMajorMinor(lineSize, sumMinor+maxMinor);
```

Comment:
After looking at the code in more details, I wonder if it wouldn't be better to update `sumMinor` inside the loop to account for the last (possibly incomplete) line? I think it would be more clear than just suddenly adding `maxMinor` here without any explanation (I know there is one in the commit message, but not in the code).